src/modules/enrollment/components/EnrollmentAssessmentActionButtons.tsx

```diff
@@ -92,9 +97,14 @@ const NewAssessmentMenu: React.FC<Props> = ({ enrollment }) => {
         : []),
       ...bottomItems,
     ];
-  }, [enrollment, getPath]);
+  }, [data, getPath]);
 
-  if (isEmpty(items)) return null;
+  if (items.length === 0)
+    return (
+      <LoadingButton loading={loading} disabled={!loading}>
```

Comment:
Nit: it would be nice if this was a skeleton while loading. Imo the loading spinner on the button evokes the idea that the button is performing some action (meaning I clicked it) since that's how we display loading submissions elsewhere.

Reply:
oh good point. Yeah skeleton would be better, I'll update it